app/models/insured/factories/self_service_factory.rb

```diff
@@ -193,14 +195,22 @@ def build_form_params
 
       def calculate_max_applicable_aptc(enrollment, new_effective_on)
         if EnrollRegistry.feature_enabled?(:temporary_configuration_enable_multi_tax_household_feature)
-          max_aptc = ::Operations::PremiumCredits::FindAptc.new.call({hbx_enrollment: enrollment, effective_on: new_effective_on}).value!
-          float_fix([max_aptc, enrollment.total_ehb_premium].min)
+          @max_aptc = fetch_aptc_value(enrollment, new_effective_on)
+          float_fix([@max_aptc, enrollment.total_ehb_premium].min)
```

Comment:
this can stay as local variable `max_aptc` once memoization implemented inside a method.